app/controllers/sessions_controller.rb

```diff
@@ -1,17 +1,18 @@
 class SessionsController < ApplicationController
   before_action :disallow_authenticated_users, only: [:new, :create]
 
-  def new
-    redirect_to home_path if user_logged_in?
-  end
-
   def create
     user = User.find_by_email(params[:email])
```

Comment:
Dynamic finders are not recommended anymore. `find_by` is recommended over them:

```ruby
user = User.find_by(email: params[:email])
```